docs/advanced/extensions.md

```diff
@@ -55,7 +55,7 @@ PLUGINS = ["saleor.extensions.plugins.custom.CustomPlugin", ]
 
 ### Configuring Plugins
 
-Saleor allows you to change a configuration of any given plugin over API. Plugin owner needs to overwrite a method to create a structure of default configuration `_get_default_configuration` and `CONFIG_STRUCTURE` . It requires an expected structure as in the following example:
+Saleor allows you to change a configuration of any given plugin over API. Plugin owner needs to overwrite a method to create a structure of default configuration `_get_default_configuration` and `CONFIG_STRUCTURE` .Plugin configuration received from API can be validated by overwriting `validate_plugin_configuration` method. It requires an expected structure as in the following example:
```

Comment:
Space removed after STRUCTURE' and added before: Plugin
_and `CONFIG_STRUCTURE`. Plugin configuration_